src/main/java/org/magicdgs/http/jsr203/HttpFileSystem.java

```diff
@@ -0,0 +1,111 @@
+/*
+ * The MIT License (MIT)
```

Comment:
Remove this license, it isn't correct and this repo will not contain license in files...

Reply:
Done.